### PR TITLE
Fix issues with app names being cut off

### PR DESCRIPTION
**Pull request by mario**
Signed-off-by: Mario Danic <mario@lovelyhq.com>

### CODE REVIEW

File: nextcloudappstore/core/static/assets/css/style.css [hunk 1]
```diff
@@ -249,6 +249,7 @@ form .text-danger {
     text-overflow: ellipsis;
     overflow: hidden;
     white-space: nowrap;
+    line-height: 1.2em;
```
Comment (MorrisJobke) [inline nextcloudappstore/core/static/assets/css/style.css:252]: For me this does not change anything, but `1.3em` does fix it. (Safari on macOS)

**Comment by mario** [comment]
OK, let's do 1.3 then.

File: nextcloudappstore/core/static/assets/css/style.css (hunk 1 above)
Comment (MorrisJobke) [inline nextcloudappstore/core/static/assets/css/style.css:252]:
```suggestion
    line-height: 1.3em;
```

**Comment by MorrisJobke** [comment]
Deployed

**Comment by mario** [comment]
@MorrisJobke hm I still see the same issue on the store. Was it really deployed? :D

**Comment by MorrisJobke** [comment]
> @MorrisJobke hm I still see the same issue on the store. Was it really deployed? :D

Correct - there was a deployment command missing :)

**Comment by mario** [comment]
Cool, please let me know when deployed :)

**Comment by MorrisJobke** [comment]
> Cool, please let me know when deployed :)

Already done since then ;)

**Comment by mario** [comment]
I still see the same issue somehow: https://apps.nextcloud.com/?search=Registration

**Comment by MorrisJobke** [comment]
Works here and the style is there:

Bildschirmfoto 2020-05-12 um 10 39 03

Maybe cached in your browser?

**Comment by mario** [comment]
Ah yup, sorry for the noise!

**Pull request merged**